packages/react-components/src/components/styles/Captions.style.ts

```diff
@@ -65,9 +65,13 @@ export const captionsBannerClassName = (formFactor: 'default' | 'compact'): ISta
 /**
  * @private
  */
-export const loadingBannerClassName = mergeStyles({
-  height: _pxToRem(100)
-});
+export const loadingBannerClassName = (formFactor: 'default' | 'compact'): IStackStyles => {
```

Comment:
Can we change the naming for these styles? I don't agree that they are a classname, we should be calling this `loadingBannerStyles` there are a lot of other classes like this.

Reply:
I agree with this as well.